trace/trace.go

```diff
@@ -49,7 +49,9 @@ func Setup(name, ver, agent, typ string, prm float64) error {
 		},
 	}
 	time.Sleep(100 * time.Millisecond)
-	metricsFactory := prometheus.New()
+	metricsFactory := prometheus.New(
```

Comment:
Question: If you pass in an empty list will it work and revert to the default prometheus one?

Reply:
Hello, that's correct! If you do not set the env var, the default prometheus buckets will be used instead.